### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by codyhazelwood** [zo0r/react-native-push-notification#49]
I'm getting a null pointer exception (and application crash) when receiving a push notification.

Changing line 35 of RNPushNotificationListenerService.java to:
```
} catch (Exception e) {
```
prevents the crash and allows the push notification to appear.

Stacktrace:
```
04-23 22:14:22.228   983  2194 D PMS     : acquireWL(b445424): PARTIAL_WAKE_LOCK  wake:*****/com.dieam.reactnativepushnotification.modules.RNPushNotificationListenerService 0x1 18597 10242 null
04-23 22:14:22.256 18597 18675 E AndroidRuntime: FATAL EXCEPTION: AsyncTask #1
04-23 22:14:22.256 18597 18675 E AndroidRuntime: Process: *****, PID: 18597
04-23 22:14:22.256 18597 18675 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at org.json.JSONTokener.nextCleanInternal(JSONTokener.java:116)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at org.json.JSONTokener.nextValue(JSONTokener.java:94)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at org.json.JSONObject.<init>(JSONObject.java:156)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at org.json.JSONObject.<init>(JSONObject.java:173)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at com.dieam.reactnativepushnotification.modules.RNPushNotificationListenerService.getPushData(RNPushNotificationListenerService.java:34)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at com.dieam.reactnativepushnotification.modules.RNPushNotificationListenerService.onMessageReceived(RNPushNotificationListenerService.java:19)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at com.google.android.gms.gcm.GcmListenerService.zzq(Unknown Source)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at com.google.android.gms.gcm.GcmListenerService.zzp(Unknown Source)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at com.google.android.gms.gcm.GcmListenerService.zzo(Unknown Source)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at com.google.android.gms.gcm.GcmListenerService.zza(Unknown Source)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at com.google.android.gms.gcm.GcmListenerService$1.run(Unknown Source)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
04-23 22:14:22.256 18597 18675 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:818)
04-23 22:15:01.187 18597 18675 D Process : killProcess, pid=18597
04-23 22:15:01.198 18597 18675 D Process : com.android.internal.os.RuntimeInit$UncaughtHandler.uncaughtException:113 java.lang.ThreadGroup.uncaughtException:693 java.lang.ThreadGroup.uncaughtException:690 

```

**Comment by zo0r** [comment]
Thats a bug introduced in https://github.com/zo0r/react-native-push-notification/commit/bdd63fae921bb31dc0cbd1cddcb530b932a4502f, [fixed here](https://github.com/zo0r/react-native-push-notification/commit/752b96fdd82de45f78e8e9d1a6e7066d83f5cac2) and published [1.0.5](https://www.npmjs.com/package/react-native-push-notification) 

Thanks for reporting!

**Comment by codyhazelwood** [comment]
Cool.  Work well now.  Thanks for the fast help and fast fix!
